### PR TITLE
Fixed duplicate names of metallic and roughness textures when assigni…

### PR DESCRIPTION
**Pull request by abdominator**
…ng textures.

### CODE REVIEW

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/Editor/GUI/Editors/EntityEditor.cpp [hunk 1]
```diff
@@ -339,13 +336,10 @@ void EntityEditor::MaterialEditor(Component::Material* material) {
...
         
         for (TextureAsset* texture : Resources().textures) {
-            if (ImGui::Selectable(texture->name.c_str()))
-                material->roughness = texture;
-
             if (ImGui::Selectable(texture->name.c_str())) {
                 if (material->roughness != Hymn().defaultRoughness)
                     Managers().resourceManager->FreeTextureAsset(material->roughness);
-                
+
```
Comment (1uk45) [inline src/Editor/GUI/Editors/EntityEditor.cpp]: New line not needed

**Comment by andens** [comment]
Reopening under the assumption that the intention was to merge. Please close again and delete branch if closing was indeed intended.

**Comment by NinjaDanz3r** [comment]
Closing since this does not seem to do anything and is incompatible with new system.

**Pull request closed without merging**